mantidproject/mantid
# Remove output workspace group before calling Fit.

Fixes #13776.

**To test**
1. Fit a few spectra.
2. Remove some of them and fit again.
3. Check that there are no unwanted workspaces left from the previous fit.
